src-v2/Judge/NTS.Judge.Blazor/Pages/Setup/RfidTags/RfidTagForm.razor

```diff
@@ -0,0 +1,50 @@
+﻿@using NTS.Judge.Blazor.Ports
+@using NTS.Domain.Setup.Entities
+
+<MudText Class="mt-2" Typo="Typo.body2">Scan Rfid Chip (optional)</MudText>
+
+<MudContainer Class="d-flex flex-row">
+    @if(Tag!=null && !_awaitingInput){
+        <MudText Class="mt-2"> Contestant Number: @Tag.Number | Tag Id: @Tag.TagId</MudText>
```

Comment:
Can't we use `ToString` here?

Reply:
We could but I realized that the Tag number is redundant since it is configured by the same form.

Reply:
`ToString` is called implicitly in the RenderTree method of every component.